src/password.php

```diff
@@ -168,23 +142,24 @@
         </div>
     </div>
 </div>
-<div class="modal" tabindex="-1" role="dialog" id="edit" data-id="">
+<div class="modal" tabindex="-1" role="dialog" id="edit" data-id="-1">
     <div class="modal-dialog">
         <div class="modal-content">
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                <h4>Edit account information</h4>
+                <h4 class="editOnly">Edit account information</h4>
+                <h4 class="addOnly">Add a new account</h4>
             </div>
             <div class="modal-body">
             <form method="post">
                 <div class="form-group">
-                    <label for="edititeminput" class="control-label">Account (Item):</label>
+                    <label for="edititeminput" class="control-label">Account (Item)</label>
                     <input class="form-control" id="edititeminput" type="text" />
                 </div>
                 <div class="form-group">
-                    <label for="edititeminputpw" class="control-label">Password:</label>
+                    <label for="edititeminputpw" class="control-label">Password</label>
                     <div class="input-group">
-                        <input class="form-control passwordText" id="edititeminputpw" type="text" autocomplete="off" placeholder="Leave blank to generate one"/>
+                        <input class="form-control passwordText" id="edititeminputpw" type="password" autocomplete="off" placeholder="Leave blank to generate one"/>
                         <span class="input-group-btn">
                             <button id="editPasswordInput" class="btn btn-warning" type="button" title="generate new password"><i class="glyphicon glyphicon-refresh"></i></button>
                             <button class="btn btn-default" type="button" id="editAccountShowPassword" title="show current password"><i class="glyphicon glyphicon-eye-open"></i></button>
```

Comment:
Show current password

Reply:
Is this the same as the one above or did you mean another line?

Reply:
Oh sorry, by typo I ment the capital letter at the beginning !
You know, just to say something because the rest was perfect ^^

Reply:
Thanks :)

Reply:
I got confused by the highlighted lines, sorry. Now I know what you meant.